akka-http-bench-jmh/src/main/scala/akka/http/impl/model/parser/UriParserBenchmark.scala

```diff
@@ -0,0 +1,35 @@
+package akka.http.impl.model.parser
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import akka.parboiled2.UTF8
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+@State(Scope.Benchmark)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@BenchmarkMode(Array(Mode.Throughput))
+class UriParserBenchmark {
+  implicit val system: ActorSystem = ActorSystem("uri-parser-benchmark")
+
+  val url = "http://any.hostname?param1=111&amp;param2=222"
```

Comment:
Could you make this a `@Param` instead so that we can test more that this one uri? There might be performance implications not just for the immediate case to be changed.

Reply:
Ok, I will also add a bigger url.